Makefile

```diff
@@ -1,8 +1,22 @@
+###########################
```

Comment:
In my opinion, any optimizations to the Makefile can be done at a later date provided the operator can be built consistently.